malariagen_data/anopheles.py

```diff
@@ -5851,7 +5851,7 @@ def plot_frequencies_map_markers(self, m, ds, variant, taxon, period, clear=True
             marker.popup = ipyleaflet.Popup(
                 child=ipywidgets.HTML(popup_html),
             )
-            m.add(marker)
+            m.add_control(marker)
```

Comment:
Thanks for the idea! Unfortunately that doesn't appear to have made any difference. I was hoping it might magically fix the 3.9.16 icon marker glitch.

I suppose a bunch of markers could/should be added as a `MarkerCluster`, i.e.:
```
from ipyleaflet import MarkerCluster
marker_cluster = MarkerCluster(
    markers=(marker1, marker2, marker3), name="marker cluster"
)
m.add_layer(marker_cluster)
```
But doesn't seem necessary (yet).

Reply:
Actually, MarkerCluster looks cool, i.e. I think it works well interactively for overlapping markers, but it still doesn't cure the icon glitch in 3.9.

Reply:
Their own example has exactly the same AwesomeIcon glitch when run on colab:
https://ipyleaflet.readthedocs.io/en/latest/layers/awesome_icon.html
<img width="672" alt="Screenshot 2023-03-25 at 14 28 27" src="https://user-images.githubusercontent.com/4256466/227723416-65820580-195c-4053-a73b-52cf13d8e162.png">

Reply:
It looks like the icon glitch appears in ipyleaflet 0.17. If you install ipyleaflet<0.17.0 then [it works](https://colab.research.google.com/drive/1K2UPG4FsbLkndb1VrAuIuNF4VHgRfgWa?usp=sharing).

Reply:
> Actually, MarkerCluster looks cool, i.e. I think it works well interactively for overlapping markers, but it still doesn't cure the icon glitch in 3.9.

FWIW I liked MarkerCluster on first sight, but have since gone off it a bit. I think the problem is that when you zoom out, it will merge multiple markers into a single marker. That merging behaviour is a bit unnecessary I think and hides information about where collections sites actually are.

Reply:
Yeah, I know what you mean, and I agree.